src/05-objects.js

```diff
@@ -22,7 +24,9 @@ function createEmptyObject() {}
  *  createObjectWithValue(19);
  *  //> { whateverKey: 19 }
  */
-function createObjectWithValue(val) {}
+function createObjectWithValue(val) {
```

Comment:
Nice work here.  Make sure we know what the ` key ` in this object is going to evaluate too in addition to its value.